### PR TITLE
Feature: Add option to have multi lines header

### PR DESCRIPTION
**Pull request by Bwaim**
- Add the new parameter `enableMultilinesHeaderRow` to have multi lines header
- Fix the position of one line header text (to test the bug put `headerRowPadding` to `0` in `activity_static.xml`, you will see that the header is not at the top)

### CODE REVIEW

**Comment by thellmund** [comment]
Please add a screenshot to this pull request to make it easier to understand what has changed.

**Comment by Bwaim** [comment]
Before :
image

After :
image

File: library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt [hunk 1]
```diff
@@ -21,14 +25,37 @@ internal class DayLabelDrawer(
...
-        val y = drawingConfig.headerTextHeight + config.headerRowPadding
 
         val textPaint = if (day.isToday) {
             drawingConfig.todayHeaderTextPaint
         } else {
             drawingConfig.headerTextPaint
         }
-        canvas.drawText(dayLabel, x, y, textPaint)
+
+        if (!config.enableMultilinesHeaderRow) {
```
Comment (thellmund) [inline library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt]: I’m not completely happy with the naming. Maybe something like `singleLineHeader`, which is set to `true` by default?

File: library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt [hunk 2]
```diff
@@ -21,14 +25,37 @@ internal class DayLabelDrawer(
...
+            val y = config.headerRowPadding.toFloat() - textPaint.ascent()
+            canvas.drawText(dayLabel, x, y, textPaint)
+        } else {
+
```
Comment (thellmund) [inline library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt]: Please remove unnecessary blank lines.

File: library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt [hunk 3]
```diff
@@ -21,14 +25,37 @@ internal class DayLabelDrawer(
...
+            val textPaint2 = TextPaint(textPaint)
```
Comment (thellmund) [inline library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt]: Please change the naming – `textPaint2` is not very descriptive.

File: library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt [hunk 4]
```diff
@@ -21,14 +25,37 @@ internal class DayLabelDrawer(
...
+            val staticLayout = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
Comment (thellmund) [inline library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt]: You could hide this logic in a method like `buildStaticLayout()`, which would make the code easier to read. Also, by importing `Build.VERSION.SDK_INT` and `Build.VERSION_CODES.M`, you can get rid of these long constants 😉

File: library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt [hunk 5]
```diff
@@ -21,14 +25,37 @@ internal class DayLabelDrawer(
...
+                                  StaticLayout.Builder
+                                      .obtain(dayLabel, 0, dayLabel.length, textPaint2
+                                              , config.totalDayWidth.toInt())
```
Comment (thellmund) [inline library/src/main/java/com/alamkanak/weekview/DayLabelDrawer.kt]: Splitting parameters should be done _after_ the comma, not before it.

**Comment by Bwaim** [comment]
I'll do this changes on monday 👍

**Comment by Bwaim** [comment]
Changes done, plus adding an example in the showing complete day example of a multi lines header.

**Comment by christian160984** [comment]
When will be available version 3.3 in gradle?

**Comment by thellmund** [comment]
I might have some time on the weekend to publish the next version.

**Comment by thellmund** [comment]
@christian160984 I’ve just released version 3.3.

**Pull request merged**